openshift-templates/automation-api-hsql/template.json

```diff
@@ -174,7 +174,7 @@
             "apiVersion": "v1",
             "kind": "RoleBinding",
             "metadata": {
-                "name": "edit"
+                "name": "jenkins_edit"
```

Comment:
I think this has the potential to class with https://github.com/openshift/origin/blob/master/examples/jenkins/jenkins-persistent-template.json#L207.

Perhaps we use `jenkins_pipeline_edit`?